aten/src/ATen/native_parse.py

```diff
@@ -413,6 +413,7 @@ def run(paths):
                 declaration['return'] = return_arguments if len(output_arguments) == 0 else output_arguments
                 declaration['variants'] = func.get('variants', ['function'])
                 declaration['requires_tensor'] = func.get('requires_tensor', False)
+                declaration['backend_select'] = func.get('backend_select', False)
```

Comment:
I was actually making a different suggestion -- that BackendSelect is a dispatch key like any of the other ones, so it should be in the dispatch table, not it's own key.